ci/ci/build.py

```diff
@@ -323,6 +340,7 @@ def build(self, batch, code, scope):
      --output 'type=image,"name={shq(self.image)},{shq(self.cache_repository)}",push=true' \
      --export-cache type=inline \
      --import-cache type=registry,ref={shq(self.cache_repository)} \
+     --import-cache type=registry,ref={shq(self.main_branch_cache_repository)} \
```

Comment:
ya it will, I checked. there will be an ERROR message in the buildkit output that it couldn't find the cache but then it will just proceed without it.